@here/harp-datasource-protocol/lib/StringEncodedNumeral.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Enumeration of supported string encoded numerals.
+ */
+export enum StringEncodedNumeral {
+    Meters,
+    Pixels,
+    HexRGB,
+    RGB,
+    HSL
+}
+
+/**
+ * Interface containing information about a [[StringEncodedNumeral]] format, component size and
+ * evaluation.
+ */
+export interface StringEncodedNumeralFormat {
+    type: StringEncodedNumeral;
+    size: number;
+    regExp: RegExp;
+}
+
+/**
+ * Array of supported [[StringEncodedNumeralFormat]]s.
+ */
+export const StringEncodedNumeralFormats: StringEncodedNumeralFormat[] = [
+    {
+        type: StringEncodedNumeral.Meters,
+        size: 1,
+        regExp: /([0-9]{1,})+(m)/
```

Comment:
No support for fractional meters and pixels?
Also, I beleive ```{1,}``` is equivalent to ```+``` in regexp.

Reply:
Fair point. Will address in next patch-set.